seed/data_importer/views.py

```diff
@@ -289,8 +290,12 @@ def create_from_pm_import(self, request):
                 'message': "Must pass pm_data in the request body."
             }, status=status.HTTP_400_BAD_REQUEST)
 
+        # base file name (will be appended with a random string to ensure uniqueness anyway, but this helps for display)
+        today_date = datetime.datetime.today().strftime('%Y-%m-%d')
+        file_name = "pm_import_%s.csv" % today_date
+
         # create a folder to keep pm_import files
-        path = os.path.join(settings.MEDIA_ROOT, "uploads", "pm_imports", "pm_import.csv")
+        path = os.path.join(settings.MEDIA_ROOT, "uploads", "pm_imports", file_name)
```

Comment:
Instead of using `pm_import.csv`, we use `pm_import_20XX-MM-DD.csv`.

The next block below then uses the FileSystemStorage class to make this unique if it is needed (if the user does two imports in the same day, for example).  It will become `pm_import_20XX-MM-DD_abcd1234.csv`.